doc/tor.1.txt

```diff
@@ -1456,6 +1454,11 @@ The following options are useful only for clients (that is, if
             error is returned: address checksum doesn't match, ed25519 public
             key is invalid or the encoding is invalid. (v3 only)
 
+          X'F7' Onion Service Introduction Timed Out
```

Comment:
We decided this together but I dont quite remember why we chose to have F2 and F7 as distinct error codes. Is it possible to document this rationale somewhere?

Reply:
https://trac.torproject.org/projects/tor/ticket/32542#comment:2